.github/workflows/master.yml

```
@@ -31,7 +31,7 @@ jobs:
 
       - uses: actions/setup-go@v5
         with:
-          go-version: 1.21
+          go-version-file: "go.mod"
```

Any specific reason for having the double quotation mark instead of single one?
```suggestion
          go-version-file: 'go.mod'
```

---

No specific reason, it's the default when linting yml in vscode but I don't have a strong opinion on this.

---

FYI, I am suggesting to change to single quotation mark to follow the same format as the GitHub Action: https://github.com/marketplace/actions/setup-go-environment#getting-go-version-from-the-gomod-file